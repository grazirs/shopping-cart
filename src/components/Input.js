import styled from "styled-components";

export const Input = styled.input `
  border-color: ${({theme}) => theme.colors.grey};
  padding: ${({theme}) => theme.space[1]};
  width: 100%;
  color: ${({theme}) => theme.colors.darkGrey};
  border: none;
  border-bottom: solid 1x ${({theme}) => theme.colors.blue};
  &:focus {
    border: none;
    border: solid 2px ${({theme}) => theme.colors.blue};
    outline: none;
    border-radius: ${({theme}) => theme.radii[1]};
  }
`
