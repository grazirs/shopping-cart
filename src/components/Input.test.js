import renderer from 'react-test-renderer';
import { Input } from "./Input";
import theme from '../theme';
import { ThemeProvider } from 'styled-components';

describe("<Input/>", () => {
  it("renders Input component correctly", () => {
    const input = renderer.create(<ThemeProvider theme={theme}><Input /></ThemeProvider>).toJSON();
    expect(input).toMatchSnapshot();
  })
});
