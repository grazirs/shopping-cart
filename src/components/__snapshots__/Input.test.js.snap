// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input/> renders Input component correctly 1`] = `
<input
  className="Input-sc-1pydahm-0 BaGZK"
/>
`;
